ci-operator/step-registry/cucushift/installer/check/azure/multidisk/cucushift-installer-check-azure-multidisk-commands.sh

```diff
@@ -0,0 +1,356 @@
+    cp_multi_disk=$(echo "${cp_multi_disk}" | jq -c -S ". +={\"security_encryption_type\":\"$(yq-go r ${INSTALL_CONFIG} 'controlPlane.platform.azure.dataDisks[0].managedDisk.securityProfile.securityEncryptionType')\"}")
+    cp_multi_disk=$(echo "${cp_multi_disk}" | jq -c -S ". +={\"security_profile_des\":\"$(yq-go r ${INSTALL_CONFIG} 'controlPlane.platform.azure.dataDisks[0].managedDisk.securityProfile.diskEncryptionSet.id')\"}")
+    if [[ "${cp_multi_disk_type}" == "user-defined" ]]; then
+        cp_mount_path="$(yq-go r ${INSTALL_CONFIG} 'controlPlane.diskSetup[0].userDefined.mountPath')"
+    elif [[ "${cp_multi_disk_type}" == "etcd" ]]; then
+        cp_mount_path="/var/lib/etcd"
+    elif [[ "${cp_multi_disk_type}" == "swap" ]]; then
+        cp_mount_path="swap"
+    fi
+    cp_multi_disk=$(echo "${cp_multi_disk}" | jq -c -S ". +={\"mount_path\":\"${cp_mount_path}\"}")
+
+    echo "
+expected multi disk options on control plane nodes:
+    disk type: ${cp_multi_disk_type}
+    disk size: $(echo ${cp_multi_disk} | jq -r '.disk_size')
+    disk caching type: $(echo ${cp_multi_disk} | jq -r '.caching_type')
+    disk lun id: $(echo ${cp_multi_disk} | jq -r '.lun_id')
+    disk storage account type: $(echo ${cp_multi_disk} | jq -r '.storage_account_type')
+    disk encryption set: $(echo ${cp_multi_disk} | jq -r '.disk_encyption_set')
+    disk security encryption type: $(echo ${cp_multi_disk} | jq -r '.security_encryption_type')
+    disk security profle des: $(echo ${cp_multi_disk} | jq -r '.security_profile_des')
+    disk mount path: $(echo ${cp_multi_disk} | jq -r '.mount_path')
+    "
+
+    master_nodes_list=$(oc get machines.machine.openshift.io -n openshift-machine-api --selector machine.openshift.io/cluster-api-machine-type=master -ojson | jq -r '.items[].metadata.name')
+    for master_node in ${master_nodes_list}; do 
+        node_json=$(mktemp)
+        echo "**********Checking data disk properties on node ${master_node}**********"
+        echo "disk settings for data disk of node:"
+        az vm show -n ${master_node} -g ${RESOURCE_GROUP} --query "storageProfile.dataDisks[?name == '${master_node}_${cp_multi_disk_name_suffix}']" -ojson | tee -a ${node_json}
+        multi_disk_check "${cp_multi_disk}" "${node_json}" || check_result=1
+
+        # check mounted path is correct on node
+        if oc debug node/${master_node} -n default -- chroot /host lsblk | grep -i "$(echo ${cp_multi_disk} | jq -r '.mount_path')" | grep "$(echo ${cp_multi_disk} | jq -r '.disk_size')G"; then
+            echo "INFO: checking disk mounted path for data disk on node passed!"
+        else
+            echo "ERROR: checking disk mounted path for data disk on node failed!"
+            oc debug node/${master_node} -n default -- chroot /host lsblk
+            check_result=1
+        fi
+
+        if [[ "${cp_multi_disk_type}" == "swap" ]]; then
+            swap_result=$(oc debug node/${master_node} -n default -- chroot /host swapon)
```

Comment:
Can we check the size of swap partition together here ?

Reply:
line258 filters the expected block devices based on mount path and disk size, here just check if swap is activated or not.